src/main/java/com/cleanroommc/modularui/value/sync/PanelSyncManager.java

```diff
@@ -42,9 +42,12 @@ public PanelSyncManager() {}
     public void initialize(String panelName, ModularSyncManager msm) {
         this.modularSyncManager = msm;
         this.panelName = panelName;
-        this.syncHandlers.forEach((mapKey, syncHandler) -> syncHandler.init(mapKey, this));
+        this.syncHandlers.forEach((mapKey, syncHandler) -> {
+            if (!syncHandler.isValid())
+                syncHandler.init(mapKey, this);
+        });
```

Comment:
If the sync handler was previously on a different panel sync manager (or still is) this might cause issues